### DISCUSSION TITLE
Failed to start Repeater ,valid config file not found

### DISCUSSION THREAD

**Discussion by Genxster1998** [Mygod/VPNHotspot#246] [Q&A]
Why won't VPN Hotspot can change wifi status automatically to turn on even when it's system app and root allowed .It always ask me to enable wifi manually when enabling repeater .whether current api doesn't start wifi Direct on enabling wifi via 3rd party access.It should be feasible right to turn on wifi automatically before turning repeater on .I am using this cause on android 10 DHCP server moved to java instead so no static IP without repeater for which static IP always is 192.168.49.*
ie. Gateway address 192.168.49.1 .

Device : Violet
Android : 10
Selinux : 1
Root Status : acquired
App ver : Latest

**Comment by Mygod** [comment]
I am not sure which is your issue. Try turning off and on Wi-Fi again.
